bot/src/bot.js

```diff
@@ -48,9 +67,15 @@ export function createBot(url, name, db) {
 
     const text = message.slice(name.length + 2);
 
-    saveUpdate(db, author, channel, text).then(
-      () => client.say(channel, `${author}: ${randElem(CONFIRMATIONS)}`)
-    );
+    if (text.includes('create a report for')) {
+      createReport(db, text).then(
+        () => client.say(channel, `${author}: ${randElem(CONFIRMATIONS)}`)
```

Comment:
could you send a special confirmation here? something to the effect of "report created". so that it's clear that it was a special command and resulted in a special behavior.